A0Auth0.podspec

```
@@ -17,5 +17,5 @@ Pod::Spec.new do |s|
   s.requires_arc = true
 
   s.dependency 'React-Core'
-  s.dependency 'Auth0', '~> 2.3'
+  s.dependency 'Auth0', '2.3'
```

```suggestion
  s.dependency 'Auth0', '2.3.2'
```
That's the last release that supports iOS 12.